contents/docs/session-replay/troubleshooting.mdx

```diff
@@ -75,6 +75,16 @@ By default, `base` and `assets` imported from `$app/paths` in Svelte use relativ
 
 For apps with SSR enabled you should set `relative` to `false` in the [`paths` object of your config](https://kit.svelte.dev/docs/configuration#paths). This ensures all asset URLs will correctly be resolved against the base URL rather than the current page.
 
+## Unable to filter by user or page properties
+
+Filtering options in session replays depend on [product analytics](/docs/product-analytics) events. 
+
+To enable useful filtering options, we recommend at the very latest:
```

Comment:
```suggestion
To enable all filtering options, we recommend at the very latest:
```

Reply:
it's not all filtering e.g. date filtering, log message filtering don't depend on events